index.opam

```diff
@@ -17,6 +17,8 @@ depends: [
   "dune"    {>= "1.11.0"}
   "fmt"
   "logs"
+  "metrics"
```

Comment:
would be nice to not depend on `metrics` at all in the index file (or if really needed, only as a test dependency)